src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/BinaryExpression.cs

```diff
@@ -1234,31 +1457,71 @@ private static BinaryExpression GetComparisonOperator(ExpressionType binaryType,
-        /// and the <see cref="BinaryExpression.Left"/> and <see cref="BinaryExpression.Right"/> properties set to the specified values.</returns>
+        /// <summary>Creates a <see cref="System.Linq.Expressions.BinaryExpression" /> that represents a conditional <see langword="AND" /> operation that evaluates the second operand only if the first operand evaluates to <see langword="true" />.</summary>
+        /// <param name="left">A <see cref="System.Linq.Expressions.Expression" /> to set the <see cref="System.Linq.Expressions.BinaryExpression.Left" /> property equal to.</param>
+        /// <param name="right">A <see cref="System.Linq.Expressions.Expression" /> to set the <see cref="System.Linq.Expressions.BinaryExpression.Right" /> property equal to.</param>
+        /// <returns>A <see cref="System.Linq.Expressions.BinaryExpression" /> that has the <see cref="System.Linq.Expressions.Expression.NodeType" /> property equal to <see cref="System.Linq.Expressions.ExpressionType.AndAlso" /> and the <see cref="System.Linq.Expressions.BinaryExpression.Left" /> and <see cref="System.Linq.Expressions.BinaryExpression.Right" /> properties set to the specified values.</returns>
+        /// <exception cref="System.ArgumentNullException"><paramref name="left" /> or <paramref name="right" /> is <see langword="null" />.</exception>
+        /// <exception cref="System.InvalidOperationException">The bitwise <see langword="AND" /> operator is not defined for <paramref name="left" />.Type and <paramref name="right" />.Type.
+        /// -or-
+        /// <paramref name="left" />.Type and <paramref name="right" />.Type are not the same Boolean type.</exception>
+        /// <remarks>The resulting <see cref="System.Linq.Expressions.BinaryExpression" /> has the <see cref="System.Linq.Expressions.BinaryExpression.Method" /> property set to the implementing method. The <see cref="System.Linq.Expressions.Expression.Type" /> property is set to the type of the node. If the node is lifted, the <see cref="System.Linq.Expressions.BinaryExpression.IsLifted" /> and <see cref="System.Linq.Expressions.BinaryExpression.IsLiftedToNull" /> properties are both <see langword="true" />. Otherwise, they are <see langword="false" />. The <see cref="System.Linq.Expressions.BinaryExpression.Conversion" /> property is <see langword="null" />.
+        /// The following information describes the implementing method, the node type, and whether a node is lifted.
+        /// #### Implementing Method
+        /// The following rules determine the implementing method for the operation:
+        /// -   If the <see cref="System.Linq.Expressions.Expression.Type" /> property of either <paramref name="left" /> or <paramref name="right" /> represents a user-defined type that overloads the bitwise `AND` operator, the <see cref="System.Reflection.MethodInfo" /> that represents that method is the implementing method.
+        /// <format type="text/markdown"><![CDATA[
+        /// > [!NOTE]
```

Comment:
There are some instances of `[!NOTE]` alone in a `<format type="text/markdown">` tag. I don't know if that's all right or if the content below the `<format>` tag is intended to be within it.

Reply:
There's a substantial number of such elements in the linq docs. Reading the [porting instructions](https://github.com/carlossanlop/DocsPortingTool/blob/main/BackportInstructions.md#instructions) I _think_ the guideline is to move embedded markdown into standalone `.md` files. 

I don't think creating a markdown file for each `[!NOTE]` section is scalable. @gewarren or @carlossanlop might have better suggestions.

Reply:
There is [some content](https://docs.microsoft.com/en-us/dotnet/api/system.linq.expressions.expression.andalso?view=net-5.0#System_Linq_Expressions_Expression_AndAlso_System_Linq_Expressions_Expression_System_Linq_Expressions_Expression_) that was dropped here.

![image](https://user-images.githubusercontent.com/24882762/115309279-e486d880-a120-11eb-952f-a624ae9e4929.png)

However, I don't think the guidelines are to move every note block to a separate file.

Reply:
This looks like an issue with the tool. Relevant docs: https://github.com/dotnet/dotnet-api-docs/blob/7305899ce8d589ed5a58281b63deb680c5e506d2/xml/System.Linq.Expressions/Expression.xml#L1198

cc @carlossanlop

Reply:
I manually fixed the truncated node elements. Turns out only the documentation for BinaryExpression was affected by the issue.